src/app/views/forgotten-password/setForgottenPasswordRequest.jsx

```diff
@@ -1,14 +1,30 @@
 import React from "react";
+import ValidateNewPassword from "../../components/validate-new-password/ValidateNewPassword";
+import PropTypes from "prop-types";
+import ButtonWithSpinner from "../../components/button/ButtonWithSpinner";
+
+const propTypes = {
+    isSubmitting: PropTypes.bool,
+    onSubmit: PropTypes.func,
+    validityCheck: PropTypes.func,
+    heading: PropTypes.string,
+    buttonText: PropTypes.string,
+    showInputError: PropTypes.bool
+};
 
 const SetForgottenPasswordRequest = props => {
-    // TODO this page is to be done in an upcoming task
     return (
         <div className="grid grid--justify-center">
             <div className="grid__col-3">
-                <h1>Request password set</h1>
+                <h1>{props.heading}</h1>
+                <form className="form" onSubmit={props.onSubmit}>
+                    <ValidateNewPassword updateValidity={props.validityCheck} inputError={props.showInputError ? " " : ""} />
```

Comment:
Aware this is a little hacky, inputError is passed down to an Input component which if it has inputError it will make the input red (errored state) but as per the UX designs, we don't want the input error text to show.

Reply:
I would investing the possibilities to remove this hack. can you maybe pass a boolean instead?

Reply:
This has now been done ^^